src/smallmap_gui.cpp

```diff
@@ -1004,6 +1005,43 @@ class SmallMapWindow : public Window {
 		}
 	}
 
+	/**
+	 * Adds industry names to the smallmap.
+	 * @param dpi the part of the smallmap to be drawn into
+	 */
+	void DrawIndustryNames(const DrawPixelInfo *dpi) const
```

Comment:
Structure follows the existing `DrawTowns(…)` method:

https://github.com/OpenTTD/OpenTTD/blob/e35680d0cbf8fe8e7c4b55729a45e9b0257f09fa/src/smallmap_gui.cpp#L988-L1006